dist/types/src/grid.d.ts

```diff
@@ -1,9 +1,10 @@
+import { AxisScale } from 'd3-axis';
 import { IAxes, IHistogramProps } from './Histogram';
 import { AnyScale } from './utils/scales';
 import { TSelection } from './utils/svg';
-export declare const makeYGridLines: (y: any, ticks?: number) => import("d3-axis").Axis<import("d3-axis").AxisDomain>;
+export declare const makeYGridLines: (y: AxisScale<any>, ticks?: number) => import("d3-axis").Axis<any>;
```

Comment:
Was this intentionally set to `any`?

Reply:
Its intentional in that the values you use on the scale can be multiple types and I'm not sure what that whole set of types is yet. I thought it better to be general than to restrict it and get potential type errors in certain charts. We can revisit this later (there's lots of improvements that can be made in the types) when things are a bit less pressing